src/jot.c

```diff
@@ -2205,22 +2208,86 @@ static int is_jottable_cmd(cmd_rec *cmd, int *cmd_ids, size_t ncmd_ids) {
   return jottable;
 }
 
+/* Here, we will return -1 if we don't explicitly know whether the given
+ * command is jottable.
+ */
+static int is_jottable_cmd_name(cmd_rec *cmd, const char **cmd_names,
+    size_t ncmd_names) {
+  register unsigned int i;
+  int jottable = -1;
+
+  for (i = 0; i < ncmd_names; i++) {
+    const char *cmd_name;
+    int negated = FALSE;
+
+    cmd_name = cmd_names[i];
+
+    if (cmd_name[0] == '!') {
+      negated = TRUE;
+      cmd_name++;
+    }
+
+    if (strcmp(cmd_name, (const char *) cmd->argv[0]) == 0) {
```

Comment:
With this implementation, we do a direct comparison of the configured commands in the "sifting" list against the command being logged/processed.  This way, the approach applies to any/all commands/requests names across protocols, and Just Works(tm) for any future names as well.